fix: let's specify how terragrunt block works in case stack isn't terragrunt (#594)
spacelift-io/user-documentation: docs/concepts/configuration/runtime-configuration/README.md

!!! warning
    You must set all necessary fields because if the `terragrunt` section is present in the config, it replaces all Terragrunt values rather than merging them.
    For example, if you set `terraform_version` only, all other fields will be reset to their defaults.

!!! info

>     Terragrunt specific parameters only work when the stack is a terragrunt stack. Adding this does not override the stack type. For example, Ansible stacks don't become Terragrunt stacks.

Reviewer: ```suggestion
    Terragrunt specific parameters only work when the stack is a Terragrunt stack. Adding this does not override the stack type. For example, Ansible stacks don't become Terragrunt stacks.
```

!!! warning
    You must set all necessary fields because if the `terragrunt` section is present in the config, it replaces all Terragrunt values rather than merging them.
    For example, if you set `terraform_version` only, all other fields will be reset to their defaults.

> !!! info
>     This parametrizes only if a stack is defined as a Terragrunt stack. Adding this does not allow us to override the stack type. For example, Ansible stacks don't become Terragrunt now.
>     It could be helpful for monorepo; it works for Terragrunt stacks only, different ones left untouched.

Reviewer 2: ```suggestion
!!! info
    Terragrunt specific parameters only work when the stack is a terragrunt stack. Adding this does not override the stack type. For example, Ansible stacks don't become Terragrunt now.
```
